core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -157,4 +157,30 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging {
       Array("--partitions", numPartitions.toString, "--replication-factor", "1", "--topic", topic, "--if-not-exists"))
     TopicCommand.createTopic(zkUtils, createNotExistsOpts)
   }
+
+  @Test
+  def testCreateAlterTopicWithRackAware() {
```

Comment:
This covers the situation where replica assignment is rack aware for alter operation. Again, I added this test since user input in the command line can change how rack aware is enforced.